src/common/base/tests/comparable-test.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {assert} from '../../../platform/chai-web.js';
+import {compareNulls} from '../comparable.js';
+
+describe('comparable', () => {
+  it('implements compareNulls', () => {
+    assert.equal(compareNulls(null, null), 0);
+    assert.equal(compareNulls(null, 'cat'), -1);
+    assert.equal(compareNulls('cat', null), 1);
```

Comment:
We could add a case where we compare the function with one argument (implicit null).

Reply:
Done.  actually implicit undefined..

Reply:
Ah, good point